Dockerfile

```diff
@@ -1,4 +1,47 @@
-FROM sismics/ubuntu-jetty:11.0.14
+FROM ubuntu:22.04
+
+# Run Debian in non interactive mode
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install Sismics repository
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates software-properties-common curl gnupg tzdata
+RUN curl -fsSL https://www.sismics.com/pgp | apt-key add -
+# RUN add-apt-repository "deb [arch=amd64] https://nexus.sismics.com/repository/apt-bionic/ bionic main"
+
+# Configure settings
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+RUN ln -fs /usr/share/zoneinfo/Europe/Paris /etc/localtime
+RUN dpkg-reconfigure -f noninteractive tzdata
+COPY etc /etc
+RUN echo "for f in \`ls /etc/bashrc.d/*\`; do . \$f; done;" >> ~/.bashrc
+RUN apt-get -y -q install vim less procps unzip wget && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && \
+    apt-get -y -q install openjdk-11-jdk && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+ENV JAVA_OPTS -Duser.timezone=Europe/Paris -Dfile.encoding=UTF-8 -Xmx1024m
+
+ENV JETTY_VERSION 11.0.14
```

Comment:
Why do you use the old version 11.0.14 from Feb 23, 2023 and not the lastest 11.0.19 from Dec 15, 2023 or even 12.0.5 from Dec 18, 2023?

Reply:
@bschlapsi4tf for this PR I am just trying to write a self contained image. I am also working on vulnerability fixes on the side. As part of it I may change this jetty version too

Reply:
I agree we should keep upgrades separated from this PR.